LunaDll/GlobalFuncs.cpp

```diff
@@ -1037,3 +1037,15 @@ void HandleEventsWhileLoading()
         lastTime = thisTime;
     }
 }
+
+std::string GetEditorPlacedItem()
+{
+    if(editorPlacedItem != "nil")
+    {
+        return (std::string)editorPlacedItem;
+    }
+    else if(editorPlacedItem == "nil")
+    {
+        return (std::string)"nil";
+    }
```

Comment:
This code seems pointlessly complicated for no reason. It's just
```C++
return (std::string)editorPlacedItem;
```
with extra steps.